### ISSUE TITLE
Fix color accessibility for 50 valued colors

### ISSUE THREAD

**Issue by maya** [uswds/uswds#2565]
### On white:

- red v
- green warm
- blue
- blue v
- violet warm v
- magenta v

### On black:

- gray warm
- gray cool

**Comment by maya** [comment]
Fixed in #2566.
